tests/core/pyspec/eth2spec/test/helpers/withdrawals.py

```diff
@@ -30,7 +30,10 @@ def set_validator_fully_withdrawable(spec, state, index, withdrawable_epoch=None
 
 def set_eth1_withdrawal_credential_with_balance(spec, state, index, balance=None, address=None):
     if balance is None:
-        balance = spec.MAX_EFFECTIVE_BALANCE
+        if is_post_electra(spec):
+            balance = spec.MIN_ACTIVATION_BALANCE
+        else:
+            balance = spec.MAX_EFFECTIVE_BALANCE
```

Comment:
I'd have to check how this helper is used but I wonder if we should keep `balance` at 32 ETH, regardless of the fork

Reply:
Let me just revert this change for now, I think proper changes should also include new tests for the cases when the withdrawal sweep is applied to compounding creds.